CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Changes Since Last Release
 
+### Changed defaults / behaviours
+
+- Add `xino=on` mount option for writable kernel overlay mount points to fix
+  inode numbers consistency after kernel cache flush (not applicable to
```

Comment:
Please could you remove the fuse-overlayfs reference. We don't use it in 3.11 at this time. Thanks.